### PR TITLE
Fixes #11: list users

### LINKED ISSUE #11: List of users in a room

### PR DESCRIPTION
**Pull request by Alir3z4**
Fixes #11

This should be enough for now, however in no case this is solid and stable.
It all depends on the cache (redis) and later I should add a "hearbeat" to know which connection is alive, although the hearbeat would add extra load since it has to be fired every N seconds. 

Anyway, this can be merged after passing the review and in another issue I can continue to make it more solid-ish.

In another pull request I will implement a solution to keep the cache fresh by each message.
So when a user (live-connection) sends a message to websocket, I renew the cache entry timeout and if the user didn't message it will automatically get removed from the cache.

I don't want to implement that into this pull request, because it will make it large and harder to review properly.

### CODE REVIEW

**Comment by Alir3z4** [comment]
This should be enough for now, however in no case this is solid and stable.
It all depends on the cache and later I should add a "hearbeat" solution to eat.

Anyway, this can be merged after passing the review and in another issue I can continue to make it more solid-ish.

**Comment by AmirSavand** [comment]
Good!

I just want others to know that I'm in the room (even tho haven't opened the chat all day) and they can message me and I'll get notified. 

If it's about being live, I can make client to send heartbeat signal to socket so server knows that the client is online or away.

So if someone joins like an hour later, he'll see that he's not alone and can be heard.

**Comment by Alir3z4** [comment]
@AmirSavand This is ready like really good.
It handles tons of new stuff now and much more solid.
Please have a quick review again and once approved, feel free to merge.

File: chat/tests/test_chat_consumer.py [hunk 1]
```diff
@@ -52,8 +59,118 @@ def create_user() -> User:
...
+    last_update: float = response['data']['users']['1']['last_update']
+    assert isinstance(last_update, float)
+    assert response == {
+        'data': {
+            'users': {
+                '1': {
+                    'name': 'Alireza Savand',
+                    'username': 'alireza',
+                    'left': False,
+                    'last_update': last_update
```
Comment (Alir3z4) [inline chat/tests/test_chat_consumer.py]: Last update can be:

* Last time messaged
* Last time did any kind of activities
* etc

For now, it means last message or last time joined

**Comment by AmirSavand** [comment]
Awesome stuff.

**Pull request merged**